index.html

```diff
@@ -1,276 +1,240 @@
+                        <li>Full Stack Dev</li>
+                        <li>2015</li>
+                      </ul>
+                    </div>
+                  </div>
+                  <p class="q-text">
+                    Experimental content creation feature that allows users 
+                    to add to an existing story over the course of a day without 
+                    spamming their friends.
+                  </p>
+                  <div class="mastery">
+                    <ul>
+                      <li>HTML</li>
+                      <li>Ruby on rails</li>
+                      <li>CSS</li>
+                      <li>JavaScript</li>
+                    </ul>
+                  </div>
+        
+                  <button class="btn btn-mastery" type="button">See Project</button>
+                </div>
+              </div>
+              <div class="title1 pot2">
+                <div class="title-img">
+                  <img src="./images/Snapshoot_Portfolio2.png" alt="image"/>
+                </div>
+                <div class="title-text">
+                  <div class="t-text">
+                    <div class="port">
+                      <h4>Facebook 360</h4>
+                      <ul class="ul">
+                        <li class="first-child">FACEBOOK</li>
+                        <li>Full Stack Dev</li>
+                        <li>2015</li>
+                      </ul>
+                    </div>
+                  </div>
+                  <p class="q-text">
+                    Exploring the future of media in Facebook's first Virtual Reality
+                    app; a place to discover and enjoy 360 photos and videos on Gear VR.
+                  </p>
+                  <div class="mastery">
+                    <ul>
+                      <li>HTML</li>
+                      <li>Ruby on rails</li>
+                      <li>CSS</li>
+                      <li>JavaScript</li>
+                    </ul>
+                  </div>
+                  <button class="btn btn-mastery" type="button">See Project</button>
+                </div>
+              </div>
+              <div class="title pot3">
+                <div class="title-img1">
+                  <img src="./images/Snapshoot_Portfolio3.png" alt="portfolio"/>
+                </div>
+                <div class="title-text1">
+                  <div class="t-text">
+                    <div class="port">
+                      <h4>Uber Navigation</h4>
+        
+                      <ul class="ul">
+                        <li class="first-child">Uber</li>
+                        <li>Lead Developer</li>
+                        <li>2018</li>
+                      </ul>
+                    </div>
+                  </div>
+                  <p class="q-text">
+                    A daily selection of privately personalized reads; no accounts
+                    or sign-ups required.
```

Comment:
DESKTOP VERSION:

- [ ] Kindly follow the spacing to the right and left of the cards as per the Figma design. As you can see there is more spacing. Make sure to reduce as per the design.
- [ ] Kindly also make sure to follow the spacing in the cards. There should be space in between the image and text.

Figma design:
<img width="404" alt="Screenshot 2022-05-05 at 1 39 10 AM" src="https://user-images.githubusercontent.com/70269507/166842836-b6a43544-4a15-46cb-b3e1-2deb50bf9dbc.png">

Your design:
<img width="1550" alt="Screenshot 2022-05-05 at 2 47 43 AM" src="https://user-images.githubusercontent.com/70269507/166842840-9687d627-afb8-4d4b-b996-39eb1f0b6000.png">